REFERENCE.md

```diff
@@ -7,8 +7,8 @@
 ### Classes
 
 * [`stdlib`](#stdlib): This module manages stdlib.
-* [`stdlib::manage`](#stdlib--manage): A simple place to define trivial resources
-* [`stdlib::stages`](#stdlib--stages): This class manages a standard set of run stages for Puppet. It is managed by
+* [`stdlib::manage`](#stdlibmanage): A simple place to define trivial resources
+* [`stdlib::stages`](#stdlibstages): This class manages a standard set of run stages for Puppet. It is managed by
```

Comment:
This the output of an old puppet-strings. I think the GitHub Action is doing something wrong here, perhaps use an outdated container.